Business/Mappings/ProductProfile.cs

```diff
@@ -0,0 +1,14 @@
+﻿using AutoMapper;
+using Entity.Concrete;
+using WebNetSample.Entity.Dtos;
+
+namespace Business.Mappings;
+
+public class MappingProfile : Profile
+{
+    public MappingProfile()
+    {
+        CreateMap<ProductDetails, ProductDetailDto>();
+        CreateMap<ProductDetailDto, ProductDetails>(); //vice versa
```

Comment:
Not really sure you need this comment. In programming it is considered that your code should be self-described and self-documented. There are some exceptions when comments should be left (like ToDo items) but not in our case. Let's remove this comment.